proxy/envoy/testdata/lds-v0-egressrule-auth.json.golden

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
This seems to be identical to lds-v0-auth.json.golden. Might be worth removing to make it clear.